docs/guides/web-ui/dashboards.md

```diff
@@ -1,111 +1,138 @@
+---
 
-Feel free to experiment with different block types and configurations to create a dashboard that suits your monitoring needs.
+## Creating custom dashboards
 
-## Choosing and Configuring Dashboard's Charts
+To create a dashboard from scratch:
 
-When creating a custom dashboard or modifying them in Logfire, you can choose from different chart types to visualize your data.
+1. Click the **+ Dashboard** button.
+2. Select the **Custom** tab.
 
-![Logfire Dashboard chart types](../../images/guide/browser-dashboard-chart-types.png)
+Custom dashboards are structured in a hierarchy: a dashboard contains one or more **panel groups**, and each group contains **panels**. Each panel, in turn, holds a specific **chart type**. By default, new dashboards start with one panel group.
 
-### Define Your Query
-In the second step of creating a chart, you need to input your SQL query. The Logfire dashboard's charts grab data based on this query. You can see the live result of the query on the table behind your query input. You can use the full power of PostgreSQL to retrieve your records.
+You can add more panel groups to better organize your dashboard. This is useful for grouping related visualizations, effectively allowing you to have multiple views within a single dashboard.
 
-![Logfire Dashboard chart query](../../images/guide/browser-dashboard-chart-sql-query.png)
+To add a new group, click the **Panel Group** button in the top right. You can name the group and set whether it should be expanded or collapsed by default when the dashboard loads.
 
-### Chart Preview and configuration
+To add a new visualization, you add a panel to a group. Click the **Panel** button in the top right. Inside each panel, you'll configure a chart and the SQL query that powers it.
 
-Based on your need and query, you need to configure the chart to visualize and display your data:
+You can rearrange and resize panels by dragging and dropping them after clicking the **Edit layout** button.
 
-#### Time Series Chart
+### Chart Types
 
-A time series chart displays data points over a specific time period.
+Logfire supports these chart types:
 
-#### Pie Chart
+| Chart Type  | Query Type         | Description                          |
+| ----------- | ------------------ | ------------------------------------ |
+| Time Series | TimeSeriesQuery    | Data points over time                |
+| Gauge       | TimeSeriesQuery    | Shows current value as a gauge       |
+| Table       | NonTimeSeriesQuery | Tabular data with rows and columns   |
+| Bar Chart   | NonTimeSeriesQuery | Comparisons across categories        |
+| Pie Chart   | NonTimeSeriesQuery | Data as slices of a circle |
+| Values      | NonTimeSeriesQuery | Values displayed as tiles |
 
-A pie chart represents data as slices of a circle, where each slice represents a category or value.
+!!! note
+    Queries with `TimeSeriesQuery` as the **Query Type** must return a timestamp column.
```

Comment:
```suggestion
    Queries with `TimeSeriesQuery` as the **Query Type** must include a timestamp in the selected columns, typically `time_bucket($resolution, start_timestamp)` when querying `records` or `time_bucket($resolution, recorded_timestamp)` when querying `metrics` - see [below](#resolution-variable). This will be used as the x-axis.
```